cloud_governance/common/mails/templates/policy_alert_agg_message.j2

```diff
@@ -66,6 +66,16 @@
 					{% else %}
 						{% if col == 'PublicCloud' %}
 							<td>{{ record[col] or 'AWS' }}</td>
+						{% elif col == 'policy' %}
+							{% if record[col] == 'ec2_stop' %}
```

Comment:
is it the right policy name 'ec2_stop', I thought we change it to 'stop_instance' ?